Sources/NIO/ByteBuffer-core.swift

```diff
@@ -87,7 +87,7 @@ private func toIndex(_ value: Int) -> Index {
 /// more bytes (octets).
 ///
 /// ### Allocation
-/// Use `ByteBufferAllocator.buffer(capacity: desiredCapacity)` to allocate a new `ByteBuffer`.
+/// Use `ByteBufferAllocator().buffer(capacity: desiredCapacity)` to allocate a new `ByteBuffer`.
```

Comment:
@ianpartridge maybe we should make it clear that you usually should not create a new `ByteBufferAllocator` for each allocation ? What about we make it `allocator.buffer(...)` ?